rudder-webapp/SPECS/rudder-webapp.spec

```diff
@@ -76,7 +76,7 @@ Requires: rsyslog
 Requires: %(../format-dependencies rpm %{real_epoch}:%{real_version} rudder-server-relay), %{apache}, %{apache_tools}, git-core, iproute, rsync, openssl, %{ldap_clients}, curl, acl
 
 # We need the PostgreSQL client utilities so that we can run database checks and upgrades (rudder-upgrade, in particular)
-Requires: postgresql >= 9.2
+Requires: postgresql >= 9.2, rudder-reports
```

Comment:
why not using format-dependencies like for relay?

Reply:
Because it's not done yet in this package and it's a little bit complicated
Moreover it's not needed since the exact version is already depended on by the rudder-server-root package

Reply:
it's done in the line just before